classes/ActionScheduler_ListTable.php

```diff
@@ -473,6 +499,24 @@ protected function row_action_run( $action_id ) {
 		$this->process_row_action( $action_id, 'run' );
 	}
 
+	/**
+	 * Force the data store schema updates.
+	 */
+	protected function recreate_tables() {
+		if ( is_a( $this->store, 'ActionScheduler_HybridStore' ) ) {
+			$store = $this->store;
+		} else {
+			$store = new ActionScheduler_HybridStore();
+		}
+		add_action( 'action_scheduler/created_table', array( $store, 'set_autoincrement' ), 10, 2 );
+
+		$store_schema  = new ActionScheduler_StoreSchema();
+		$logger_schema = new ActionScheduler_StoreSchema();
+		$store_schema->register_tables( 'yes' );
+		$logger_schema->register_tables( 'yes' );
```

Comment:
Nitpick: can we make this argument easier to grok from reading the call, or at least switch this to using a boolean?

See: [Self-Explanatory Flag Values for Function Arguments](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#self-explanatory-flag-values-for-function-arguments)

Reply:
I update the instructions & flag type.